mne_bids_pipeline/_config.py

```diff
@@ -1611,7 +1611,7 @@
 Must contain at least two elements. By default, 5 equally-spaced bins are
 created across the non-negative time range of the epochs.
 All specified time points must be contained in the epochs interval.
-If `None`, do not perform **time-frequency** analysis, and only run CSP on
+If an empty list, do not perform **time-frequency** analysis, and only run CSP on
```

Comment:
If we keep this new feature, can you add it to the changelog please? (list of enhancements)